### PR TITLE
Add netlify-identity submodule

### PR DESCRIPTION
**Pull request by pleasantone**
Website needs to be updated to document this submodule, and the two parameters that may be overridden:

```
[params.decap_cms._netlify_identity]
admin_path = "/admin/"
js_url = "https://identity.netlify.com/v1/netlify-identity-widget.js"
```

### CODE REVIEW

**Comment by pleasantone** [comment]
Ready for your review and merge @razonyang -- thank you!

File: modules/netlify-identity/layouts/partials/hugopress/modules/decap-cms-netlify-identity/hooks/body-end.html
Comment (razonyang) [inline modules/netlify-identity/layouts/partials/hugopress/modules/decap-cms-netlify-identity/hooks/body-end.html:1]: > That bit of code is required to redirect the response from the netlify identity server from / to /admin (or wherever you are going to base it). If no netlify response, it does nothing.

I'm still be confused by this, are you intend to force users to login into Decap CMS when they're visiting the homepage? For example, _visit `/`_ -> _not log in_ -> _redirect to Decap CMS dashboard_.

**Comment by razonyang** [comment]
Sorry for the delay response, just set up a demo on Netlify to see how it work.

**Comment by pleasantone** [comment]
No, that code only gets executed in response to the netlify server, which returns to the homepage, and really needs to go to the admin page. Someone who is not editing using the CMS will never know that it’s there

**Comment by razonyang** [comment]
> No, that code only gets executed in response to the netlify server, which returns to the homepage, and really needs to go to the admin page. Someone who is not editing using the CMS will never know that it’s there

I took a look at the identity widget library docs, that code listend the `login` event, so it seems that you've a login button on homepage, redirect to `/admin` after user login?

**Comment by pleasantone** [comment]
For me the login button only appears on the admin page, but that code doesn’t know about admin so it returns to /, and without it, I never complete the login.
I don’t get any login button visible on root.


From decap docs:
> When a user logs in with the Netlify Identity widget, an access token directs to the site homepage. In order to complete the login and get back to the CMS, redirect the user back to the /admin/ path. To do this, add the following script before the closing body tag of your site's main index page…


https://decapcms.org/docs/choosing-a-backend/

**Comment by razonyang** [comment]
> When a user logs in with the Netlify Identity widget, an access token directs to the site homepage. In order to complete the login and get back to the CMS, redirect the user back to the /admin/ path. 

I get it, thanks for the references.

**Review by razonyang — APPROVED** [review]

**Comment by razonyang** [comment]
Merged and drafted releases, thank you so much.

**Comment by pleasantone** [comment]
Thank you, I learned a lot from you!

**Pull request merged**